docs/Performance.md

```diff
@@ -1 +1,41 @@
 # Performance
+
+## Logs
+
+### CPU usage guidelines
+
+#### Setup
+
+The following benchmark has been compiled on an Amazon `m4.large`
+instance (which has 2 CPU cores and 8 GB of memory available).
+
+It can be used when estimating the required CPU resources for logs collection
+using [`filelogreceiver`][filelogreceiver].
+
+[filelogreceiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver
+
+#### Benchmark
+
+Events Per Second (EPS) achieved for a particular average message size and CPU usage.
+
+|                       | 100B | 512B  | 1KB  | 5KB  | 10KB |
+|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
+| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   |
+|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
+|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
+|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
+|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
```

Comment:
Is 1400 correct value. What is the reason the EPS is so low?

Reply:
It's not, it should be 14000.